docs/administration/configuration/configuring-interactive-sessions/index.md

```diff
@@ -0,0 +1,45 @@
+# Configuring interactive sessions
+
+## Inactive interactive sessions cleanup
+
+### Automatic closure
+
+By default, interactive sessions in REANA are kept open indefinitely. However, administrators
+have the flexibility to configure the maximum inactivity period after which interactive
+sessions will be automatically closed. This helps optimize resource utilization and ensure that unused sessions
+are not consuming unnecessary resources.
+
+The automatic closure of interactive sessions in REANA can be customized by configuring
+[Helm values](https://github.com/reanahub/reana/tree/master/helm/reana):
+
+- `interactive_sessions.maximum_inactivity_period`:
+  This option allows you to set a limit (in days) for the maximum inactivity period of interactive sessions after which they will be closed.
+  If this option is set to `forever`, interactive sessions will be kept open indefinitely unless manually closed.
+
+- `interactive_sessions.cronjob_schedule`:
+  This option allows you to define how often the cleanup process should be scheduled, using a [cron expression](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#schedule-syntax).
+
+For example, this is how you would configure the Helm chart to make REANA automatically close
+interactive sessions that have been inactive for more than 7 days, and perform the cleanup
+check every day at 3 AM:
+
+```{.yaml .copy-to-clipboard}
+interactive_sessions:
+    maximum_inactivity_period: 7
+    cronjob_schedule: "0 3 * * *"
+```
+
+### Manual closure
```

Comment:
This section could be removed, since this is just a tip for admins.

We could actually transform it into a `!!! note` so that it would stand out after the example. The text could be simplified to read:

- Note that as an administrator you can also force the manual closure of opened interactive sessions regardless of the value of `interactive_sessions.maximum_inactivity_period` by means of the `reana-admin` tool. For example, to close any interactive session that has been inactive since 30 days, you would run: ...

We could use several such `kubectl` tips throughout the admin-related documentation.

Reply:
Yes, using "tip" sounds nice, it would suit well these "howto" notes, and would differentiate them nicely from the "information" notes...